sdk/search/test.yml

```diff
@@ -0,0 +1,14 @@
+trigger: none
+
+jobs:
+    - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+      parameters:
+          ServiceDirectory: search
+          DisableAzureResourceCreation: true
+          Timeout: 120
```

Comment:
Given that we've done a live test run about how long did it take to complete? We don't want to have timeouts that are too long in cases where tests deadlock and hold a DevOps VMs for longer than needed. Is it possible to use the default timeout of 60 minutes for the Search live tests?

Reply:
Search is depending storage currently, so 60 minute usually cannot finish search tests. Besides of that, search also takes long time to run, which might need to improve in future.